wiki/BanchoBot/en.md

```diff
@@ -1,28 +1,164 @@
+
+You can also use the `/bb` in-game client command to automatically open a BanchoBot tab and send the command immediately.
+
+*Notice: The `SEARCH` command is not included as it is not working.*
+
+*Note: The user argument is case-insensitive, as usernames are case-insensitive.*
+
+### HELP
+
+```
+!HELP
+```
+
+Help shows the list of available commands.
+
+#### Usage Example
+
+*Notice: The `user` parameter for the `REPORT` command is missing.*
+
+```
+pippi: !help
+BanchoBot: Standard Commands (!COMMAND or /msg BanchoBot COMMAND):
+BanchoBot: SEARCH <query>|next|prev
+BanchoBot: WHERE <user>
+BanchoBot: STATS <user>
+BanchoBot: FAQ <item>|list
+BanchoBot: REPORT <reason> - call for an admin
+BanchoBot: REQUEST [list] - shows a random recent mod request
+BanchoBot: ROLL <number> - roll a dice and get random result from 1 to number(default 100)
+```
+
+### ROLL
+
+```
+!ROLL <number>
+```
+
+Get a random number from 1 to the selected number (default is 100).
+
+#### Usage Example
+
+```
+pippi: !roll 1000
+BanchoBot: pippi rolls 109 point(s)
+```
+
+### STATS
+
+```
+!STATS <user>
+```
+
+Displays the entered user's stats. The resulting stats depends on the mode they are playing or last played.
+
+*Note: BanchoBot will not tell you which game mode the stats came from.*
+
+*Notice: BanchoBot will respond with `User not found` if the user has never played a game, even if the user does exist.*
+
+#### Usage Examples
+
+```
+pippi: !stats peppy
+BanchoBot: Stats for peppy:
+BanchoBot: Store: 422,187,979 (#94718)
+BanchoBot: Plays: 7149 (lv65)
+BanchoBot: Accuracy: 87.29%
+```
+
+```
+pippi: !stats Ephemeral:
+BanchoBot: Stats for Ephemeral is Playing:
+BanchoBot: Store: 2,384,155,536 (#33697)
+BanchoBot: Plays: 14054 (lv96)
+BanchoBot: Accuracy: 94.19%
+```
+
+### WHERE
+
+```
+!WHERE <user>
+```
+
+Displays the entered user's current location.
+
+*Note: This only shows the user's country.*
+
+#### Usage Example
+
+```
+pippi: !where Ephemeral
+BanchoBot: Ephemeral is in Australia
+```
+
+### FAQ
+
+```
+!FAQ <entry>
+!FAQ list
+```
+
+Displays the message of the entry. You can use the `list` argument to show all available entries. When a moderator uses this in a public channel, the response will be outputted to that channel. When normal users use this in a public channel, the response will be outputted though BanchoBot's PM.
+
+#### Usage Examples
+
+```
+pippi: !faq peppy
+BanchoBot: peppy is the lead developer and indeed, the creator of osu! and handles most of the project himself.
+```
+
+```
+Tama: 你好
+Yuzu: !faq chinese
+BanchoBot: 中文用户请点击 #chinese 以进入中文频道进行交流。
+BanchoBot: #osu 是英文专属频道，如果接获此讯息后继续在 #osu 内以中文交谈，管理员有权利禁言。
+Tama: 谢谢
+```
+
+### REPORT
+
+*Notice: If reporting a moderator, email [support@ppy.sh](mailto:support@ppy.sh) for moderator complaints.*
```

Comment:
If you want to report a moderator, send an  e-mail  to [support@ppy.sh](mailto:support@ppy.sh) for moderator complaints.